MdeModulePkg/Library/VariablePolicyLib/VariablePolicyUnitTest/VariablePolicyUnitTest.c

```diff
@@ -2435,6 +2435,1438 @@ ShouldBeAbleToDumpThePolicyTableAfterDisabled (
   return UNIT_TEST_PASSED;
 }
 
+/**

```

Comment:
Probably all of the unit test commits can be squashed.

Reply:
I suggest leaving this one separated out. I'm planning to upstream it and it would be easier to compare in the future. It's also already fairly large.

Reply:
To follow up, I created the upstream PR in edk2/5930. I'm reworking it now for ECC issues. Leaving it separated here should match up cleanly.